portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,11 +44,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get response from the form
+    //Get response from the form
     String text = request.getParameter("text-input");
     if (text != null && !text.isEmpty()) {
       myComments.add(text);
     }
+    Entity comment = new Entity("Comment");
+    comment.setProperty("text", text);
```

Comment:
nit - looks like the guide is pushing for you to store the timestamp as well (which will help with loading in the next PR).

Reply:
ah definitely a great note-will add that in!